Source/WebCore/Modules/compression/CompressionStreamEncoder.h

```diff
@@ -25,42 +25,44 @@
 #pragma once
 
 #include "BufferSource.h"
+#include "CompressionStream.h"
 #include "ExceptionOr.h"
 #include "Formats.h"
+#include "ZStream.h"
 #include <JavaScriptCore/Forward.h>
 #include <wtf/RefCounted.h>
 #include <wtf/RefPtr.h>
 #include <wtf/Vector.h>
-#include <zlib.h>
 
 namespace WebCore {
 
 class CompressionStreamEncoder : public RefCounted<CompressionStreamEncoder> {
 public:
-    static Ref<CompressionStreamEncoder> create(unsigned char format)
-    { 
+    static ExceptionOr<Ref<CompressionStreamEncoder>> create(unsigned char formatChar)
```

Comment:
Why does this function take unsigned char, and not Formats::CompressionFormat?

Reply:
That's a result of weird stream JS stuff.  I think we should address that in a separate PR.

Reply:
oh, I see.